README.md

```diff
@@ -13,6 +13,19 @@ npm install
 npm start
 ```
 
+## Tasks
+
+- `npm start` Run web server.
```

Comment:
What about calling it `dev` instead since that's what we called that task in Grunt/Gulp?